cis-benchmarks/README.md

```diff
@@ -0,0 +1,29 @@
+# CIS Benchmarks
+
+This plugin utilizes the [kube-bench][kubebench] implementation of the [CIS security benchmarks][cis]. It is technically two plugins; one to run the checks on the master nodes and another to run the checks on the worker nodes.
+
+## Usage
+
+To run this plugin, run the following command:
+
+```
+sonobuoy run
+--plugin https://raw.githubusercontent.com/johnSchnake/sonobuoy-plugins/cis-benchmarks/cis-benchmarks/kube-bench-plugin.yaml --plugin https://raw.githubusercontent.com/johnSchnake/sonobuoy-plugins/cis-benchmarks/cis-benchmarks/kube-bench-master-plugin.yaml 
+```
+
+## Assumptions
```

Comment:
I don't know if you want to include it in the assumptions section, or in a requirements section, but this README should include that Sonobuoy v0.16.4 is required due needing the node affinity support.

Reply:
Ooo yea and the URL for the `--plugin` is new too.